python/cuspatial/benchmarks/pytest.ini

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
This file is probably not needed until we introduce cuDF benchmark suite.

Reply:
`pytest.ini` is necessary to tell `pytest-bechmark` which files to run.